### PR TITLE
Fixed img bug; updated formatting; ran thru linter

### PR DESCRIPTION
**Pull request by tristandenyer**
Was working with @aamir-ahmed on edits to this page to fix some image and formatting bugs. When I went to verify the latest changes, there was one small image bug. So, I fixed it.

While in there, I noticed my Markdown linter picked up on numerous small issues also causing some not-so-obvious formatting issues.

As I was checking the page, I noticed that the steps were hard to distinguish, so I improved them for 1) readability, and 2) adding in anchor tags for sharing.

**Things to look for:**
— all images now show (no raw code showing for images)
— no codeblocks present
— doc starts with an H2, steps are H3s
— converted ordered list to H3s to improve readability
— prepended each step with unique section word to 1) make anchor tagging unique (i.e. Okta - Step 2, Postman - Step 4), and 2) help the user know which site/app they should be in for that step
— removed superfluous spaces

Original error/bug:
Screen Shot 2019-07-18 at 8 03 56 AM

### CODE REVIEW

**Comment by kaustavdm** [comment]
@tristandenyer These changes look good. Can you fix the merge conflicts and update this branch?

**Comment by ArifPMan** [comment]
@kaustavdm This includes the fix you've mentioned. It makes this PR obsolete. We can get rid of this PR, no need to merge.

**Comment by kaustavdm** [comment]
@ArifPMan Sorry about the confusion. Re-confirming: Do you mean keep this PR and close #1696?

**Comment by ArifPMan** [comment]
@kaustavdm Yes, keep this and close #1696

**Comment by kaustavdm** [comment]
Rebased `master` and fixed merge conflicts.

File: v6/enterprise/sso/saml_okta.md [hunk 1]
```diff
@@ -16,138 +16,148 @@ This section describes the following two topics:
...
 
-[![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-New-Integ2A.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-New-Integ2A.png)
-
-4. Now, go to the Postman [Edit Team Details](https://go.postman.co/settings/team/general) page. To update the identity provider details, navigate to *Authentication -> <My_Okta_Integration_Name>* and click **Edit**. Next, click **Proceed**. Ensure, you are in the following screen after the completion of this step:
+### Postman - Step 4
 
+Now, go to the Postman [Edit Team Details](https://go.postman.co/settings/team/general) page. To update the identity provider details, navigate to *Authentication -> <My_Okta_Integration_Name>* and click **Edit**. Next, click **Proceed**. Ensure, you are in the following screen after the completion of this step:
 [![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/ENT-identity-provider-details.png)
```
Comment (kaustavdm) [inline v6/enterprise/sso/saml_okta.md]: Points to a different URL in the hyperlink than the image source.

File: v6/enterprise/sso/saml_okta.md [hunk 2]
```diff
@@ -16,138 +16,148 @@ This section describes the following two topics:
...
+### Okta - Step 10
+
+Copy the **Identity Provider Single Sign-On URL**, **Identity Provider Issuer** and **X.509 Certificate** from the below screen.
+[![okta identity provider](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider_updated.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider.png)
+
+And paste them in the corresponding sections of the **Identity Provider Details** screen as shown below:
 [![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details3.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details3.png)
 
 Once you fill-in the details, click the **Generate relay/Regenerate relay** button to create a parameter to send with a SAML response in an IDP-initiated single sign-on. Copy the **relay state** and paste it in the following screen:
+[![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-Relay-State.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details3.png)
```
Comment (kaustavdm) [inline v6/enterprise/sso/saml_okta.md]: Points to a different URL in the hyperlink than the image source.

File: v6/enterprise/sso/saml_okta.md [hunk 3]
```diff
@@ -16,138 +16,148 @@ This section describes the following two topics:
...
-[![okta identity provider](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider1.png)
+The **View Setup Instructions** screen comes populated with values that you should copy and paste in the **Identity Provider Details** section.
 
-And paste them in the corresponding sections of the **Identity Provider Details** screen as shown below:
+**Note:** You will need to come back to this screen to paste the value in *Default Relay State* that you will generate from the Postman's **Identity Provide Details** section.
 
+### Okta - Step 10
+
+Copy the **Identity Provider Single Sign-On URL**, **Identity Provider Issuer** and **X.509 Certificate** from the below screen.
+[![okta identity provider](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider_updated.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider.png)
```
Comment (kaustavdm) [inline v6/enterprise/sso/saml_okta.md]: Points to a different URL in the hyperlink than the image source.

File: v6/enterprise/sso/saml_okta.md [hunk 4]
```diff
@@ -16,138 +16,148 @@ This section describes the following two topics:
...
 
-6. Under the second step “Configure SAML”, section A “SAML Settings”, enter the Postman service provider details which can be found on the Postman [Edit Team Details](https://go.postman.co/settings/team/general) page. To update the identity provider details, navigate to *Authentication -> <My_Okta_Integration_Name>* and click **Edit**. Next, click **Proceed**. Ensure, you are in the following screen after the completion of this step:
+### Okta - Step 6
 
+Under the second step “Configure SAML”, section A “SAML Settings”, enter the Postman service provider details which can be found on the Postman [Edit Team Details](https://go.postman.co/settings/team/general) page. To update the identity provider details, navigate to *Authentication -> <My_Okta_Integration_Name>* and click **Edit**. Next, click **Proceed**. Ensure, you are in the following screen after the completion of this step:
 [![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/ENT-identity-provider-details.png)
```
Comment (kaustavdm) [inline v6/enterprise/sso/saml_okta.md]: Points to a different URL in the hyperlink than the image source.

File: v6/enterprise/sso/saml_okta.md [hunk 5]
```diff
@@ -16,138 +16,148 @@ This section describes the following two topics:
...
 The **View Setup Instructions** screen comes populated with values that you should copy and paste in the **Identity Provider Details** section.
 
 **Note:** You will need to come back to this screen to paste the value in *Default Relay State* that you will generate from the Postman's **Identity Provide Details** section.
 
-6. Copy the **Identity Provider Single Sign-On URL**, **Identity Provider Issuer** and **X.509 Certificate** from the below screen.
+### Postman - Step 6
 
-[![okta identity provider](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider1.png)
+Copy the **Identity Provider Single Sign-On URL**, **Identity Provider Issuer** and **X.509 Certificate** from the below screen.
+[![okta identity provider](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider_updated.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/okta_identity_provider.png)
```
Comment (kaustavdm) [inline v6/enterprise/sso/saml_okta.md]: Points to a different URL in the hyperlink than the image source.

File: v6/enterprise/sso/saml_okta.md [hunk 6]
```diff
@@ -16,138 +16,148 @@ This section describes the following two topics:
...
 And paste them in the corresponding sections of the **Identity Provider Details** screen as shown below:
-
 [![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details3.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details3.png)
 
-Once you fill-in the details, click the **Generate relay/Regenerate relay** button to create a parameter to send with a SAML response in an IDP-initiated single sign-on. Copy the **relay state** and paste it in the following screen:
+### Postman - Step 7
 
-[![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-Relay-State1A.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details1A.png)
+Once you fill-in the details, click the **Generate relay/Regenerate relay** button to create a parameter to send with a SAML response in an IDP-initiated single sign-on. Copy the **relay state** and paste it in the following screen:
+[![details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-Relay-State.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Okta-IDP-Details3.png)
```
Comment (kaustavdm) [inline v6/enterprise/sso/saml_okta.md]: Points to a different URL in the hyperlink than the image source.

**Comment by kaustavdm** [comment]
@tristandenyer Does #1726 invalidate this PR?

**Comment by kaustavdm** [comment]
@tristandenyer Looks good. Merge it if it is ready.

**Pull request merged**